### PR TITLE
feat: notion calendar

### LINKED ISSUE #13298: [CAL-2958] cron to notion calendar
null

<sub>[CAL-2958](https://linear.app/calcom/issue/CAL-2958/cron-to-notion-calendar)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
CleanShot 2024-01-18 at 10 49 12@2x

### CODE REVIEW

**Review by zomars — APPROVED** [review]

**Review by zomars — DISMISSED** [review]
We need to come up with a SQL migration to h handle the name change in DB as well

**Comment by PeerRich** [comment]
> We need to come up with a SQL migration to h handle the name change in DB as well

ohhh i see.

maybe we just keep the slug the same and just change display name and screenshots etc?

File: packages/app-store/cron/config.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "/*": "Don't modify slug - If required, do it using cli edit command",
-  "name": "Cron",
+  "name": "Notion Calendar",
   "slug": "cron",
-  "type": "cron_other",
-  "logo": "logo.png",
+  "type": "cron_calendar",
+  "logo": "icon.svg",
```
Comment (PeerRich) [inline packages/app-store/cron/config.json:6]:
```suggestion
  "logo": "logo.png",
```

File: packages/app-store/cron/config.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
   "/*": "Don't modify slug - If required, do it using cli edit command",
-  "name": "Cron",
+  "name": "Notion Calendar",
   "slug": "cron",
-  "type": "cron_other",
-  "logo": "logo.png",
+  "type": "cron_calendar",
```
Comment (PeerRich) [inline packages/app-store/cron/config.json:5]:
```suggestion
  "type": "cron_other",
```

**Review by emrysal — APPROVED** [review]
looks good to me

**Pull request merged**